lib/ddtrace/contrib/redis/vendor/resolver.rb

```diff
@@ -56,7 +56,7 @@ def _parse_options(options)
             defaults = DEFAULTS.dup
             options = options.dup
 
-            defaults.keys.each do |key|
+            defaults.each_key do |key|
```

Comment:
Glad to see this works all the way back to Ruby 2.0.

Reply:
Indeed. I did check ahead just in case :)